contrib/oc-get-customization.sh

```diff
@@ -0,0 +1,44 @@
+# shellcheck shell=bash
```

Comment:
We should use `set -euo pipefail`

Reply:
As long as this script should be invoked via `source contrib/oc-get-customization.sh` to export `BRIDGE_CONFIG`, this doesn't work, because each error while close your current shell.

If we want that the user can just run `./contrib/oc-get-customization.sh` (or maybe `./hack/get-customization.sh`) and configure the config **manually** (`bin/bridge -config ...`), we can active this options. I would prefer to keep this for now and that we meet with Christian and Jakub to talk about some cleanups?